### ISSUE TITLE
Toolbar not responding properly to window width reduction

### ISSUE THREAD

**Issue by Mxchaeltrxn** [mayeaux/nodetube#164]
When the width of the page is 1000, it looks like this. 
<img width="999" alt="image" src="https://user-images.githubusercontent.com/34886045/74001396-89574300-49bf-11ea-8cc3-208f41479c29.png">

**Comment by mayeaux** [comment]
Yeah the media queries aren't perfect. Hopefully someone who really lives for frontend can take a look at that and also the way the text breaks in the little info sections below (the way instance breaks onto the next line like that). 

I'm more of a backend guy personally.
